tests/integration/test_tostac.py

```diff
@@ -0,0 +1,61 @@
+import json
+import shutil
+from functools import partial
+from pathlib import Path
+from pprint import pformat
+
+import pytest
+from deepdiff import DeepDiff
+
+from eodatasets3.scripts import tostac
+from tests.integration.common import run_prepare_cli
+
+TO_STAC_DATA: Path = Path(__file__).parent.joinpath("data/tostac")
+ODC_METADATA_FILE: str = "ga_ls8c_ard_3-1-0_088080_2020-05-25_final.odc-metadata.yaml"
+STAC_TEMPLATE_FILE: str = "ga_ls_ard_3_stac_item.json"
+STAC_EXPECTED_FILE: str = "ga_ls8c_ard_3-1-0_088080_2020-05-25_final.stac-item_expected.json"
+
+deep_diff = partial(DeepDiff, significant_digits=6)
+
+
+def test_tostac(input_doc_folder: Path):
+    input_metadata_path = input_doc_folder.joinpath(ODC_METADATA_FILE)
+    assert input_metadata_path.exists()
+
+    input_template_path = input_doc_folder.joinpath(STAC_TEMPLATE_FILE)
+    assert input_template_path.exists()
+
+    run_prepare_cli(
```

Comment:
Why are we writing files here? Wouldn't it be better doing this in memory?

Reply:
To have complete test of `eo3-to-stac` which takes metadata files as input.